### ISSUE TITLE
object detection label map

### ISSUE THREAD

**Issue by shzygmyx** [tensorflow/models#1906]
default label map config in object_detection.data label_map.pbtxts starts from index0, however, the program results in 'Label map ids should be >= 1'.
Change the label map id start from 1 could fix the bug.
For example,  the id in pet_label_map.pbtxt are from 0 to 37. Change the id of class 'none_of_the_above' to 38, then the bug should be fixed

**Comment by AruniRC** [comment]
@shzygmyx Facing the same issue when trying to create new TF records for PASCAL dataset:

```
python create_pascal_tf_record.py --data_dir=VOCdevkit \
>     --year=VOC2012 --set=train --output_path=pascal_train.record
```
> Traceback (most recent call last):
>   File "create_pascal_tf_record.py", line 183, in <module>
>     tf.app.run()
>   File "/home/aroychow/anaconda3/envs/tensorflow2_cpu/lib/python2.7/site-packages/tensorflow/python/platform/app.py", line 48, in run
>     _sys.exit(main(_sys.argv[:1] + flags_passthrough))
>   File "create_pascal_tf_record.py", line 158, in main
>     label_map_dict = label_map_util.get_label_map_dict(FLAGS.label_map_path)
>   File "/mathworks/devel/sandbox/aroychow/tensorflow_detector_2/models/object_detection/utils/label_map_util.py", line 136, in get_label_map_dict
>     label_map = load_labelmap(label_map_path)
>   File "/mathworks/devel/sandbox/aroychow/tensorflow_detector_2/models/object_detection/utils/label_map_util.py", line 123, in load_labelmap
>     _validate_label_map(label_map)
>   File "/mathworks/devel/sandbox/aroychow/tensorflow_detector_2/models/object_detection/utils/label_map_util.py", line 36, in _validate_label_map
>     raise ValueError('Label map ids should be >= 1.')
> **ValueError: Label map ids should be >= 1.**

**Comment by korrawat** [comment]
The label maps are in the `object_detection/data` directory, which contains [`pet_label_map.pbtxt`](https://github.com/tensorflow/models/blob/master/object_detection/data/pet_label_map.pbtxt) and [`pascal_label_map.pbtxt`](https://github.com/tensorflow/models/blob/master/object_detection/data/pascal_label_map.pbtxt).

This is due to the [change](https://github.com/tensorflow/models/pull/1861/commits/88a05515fcc13d6a6fb8224b5c4465414d0d53c2#diff-aae5c755e54e3a4e52bc33af720b57c7) in #1861. @derekjchow How should the two label maps be fixed? Get rid of class 0 ("none of the above") or change its id to n+1?

**Comment by shzygmyx** [comment]
@AruniRC  I've updated my comment, hope it helps.

**Comment by derekjchow** [comment]
@korrawat correct, the label map IDs should have the 0 class removed. I thought I had removed them in PR #1861, but it seems it wasn't the case.

PR #1903 should address this issue.

**Comment by derekjchow** [comment]
Fixed by #1903
